raiden/api/python.py

```diff
@@ -1764,6 +1765,38 @@ def save_light_client(self,
 
         return result
 
+    def store_rif_comms_light_client(self, address):
```

Comment:
I think this one and the `store_matrix_light_client` method are the same code but you just put none on some of the parameters to save, maybe you could put all this code in one method only and then put the parameters like optional? so you don't repeat the code just for a few params.

Reply:
The matrix one uses a lot of matrix server related variables, i.e

```
 pubhex = self.raiden.config["pubkey"].hex()
 encrypt_signed_password = encrypt(pubhex, signed_password.encode())
 encrypt_signed_display_name = encrypt(pubhex, signed_display_name.encode())
 encrypt_signed_seed_retry = encrypt(pubhex, signed_seed_retry.encode())
```

but.. i dont like the idea of passing None, i think i can improve that

Reply:
Yeah what i suggest was that on those ocurrences just do a check and if it's not None then you do that otherwise you just pass the value from the parameter directly to the save method, i just don't like the duplicated code but anyway is just a detail.